Cannot check style on Java 11

Sorry if I have missed something obvious but I am struggling to get this extension to work. Not sure if it's something I've got wrong or if there's a genuine issue.

### env info

OS: macOS High Sierra 10.13.6 
VSCode version: 1.31.1
Language Support for Java version: 0.39.0
Checkstyle for Java version: 1.0.0

```
Jeds-MacBook-Pro:java-hello-world jedthompson$ java -version
openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)
```

`JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk-11.0.2.jdk/Contents/Home`

![screen shot 2019-02-27 at 20 43 03](https://user-images.githubusercontent.com/25291974/53521594-7b6f2180-3ad0-11e9-8ce3-c4be0a0a526f.jpg)

### Steps to reproduce

Clone this [java-hello-world](https://github.com/jedster1111/java-hello-world) project.
Only have Language Support for Java by Red Hat and Checkstyle for Java installed.
Set the CheckStyle configuration to Google's Style, Sun's Style or the file in `config/checkstyle.xml`.
Open `HelloWorld.java`.

### error messages

Checkstyle output:
```
{"message":"cannot initialize module FileTabCharacter - Unable to instantiate 'FileTabCharacter' class, it is also not possible to instantiate it as .FileTabCharacter, FileTabCharacterCheck, .FileTabCharacterCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly."}
```

Language Support for Java output:
```
[Error - 20:57:01] 27 Feb 2019, 20:57:01 Problems occurred when invoking code from plug-in: "org.eclipse.jdt.ls.core".
cannot initialize module FileTabCharacter - Unable to instantiate 'FileTabCharacter' class, it is also not possible to instantiate it as .FileTabCharacter, FileTabCharacterCheck, .FileTabCharacterCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly.
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module FileTabCharacter - Unable to instantiate 'FileTabCharacter' class, it is also not possible to instantiate it as .FileTabCharacter, FileTabCharacterCheck, .FileTabCharacterCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly.
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:463)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:198)
	at com.shengchen.checkstyle.runner.CheckstyleRunner.check(CheckstyleRunner.java:75)
	at com.shengchen.checkstyle.runner.handler.DelegateCommandHandler.executeCommand(DelegateCommandHandler.java:37)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler$1.run(WorkspaceExecuteCommandHandler.java:151)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler.executeCommand(WorkspaceExecuteCommandHandler.java:141)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$4(JDTLanguageServer.java:451)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$26(JDTLanguageServer.java:793)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:479)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Unable to instantiate 'FileTabCharacter' class, it is also not possible to instantiate it as .FileTabCharacter, FileTabCharacterCheck, .FileTabCharacterCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly.
	at com.puppycrawl.tools.checkstyle.PackageObjectFactory.createModule(PackageObjectFactory.java:210)
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:453)
	... 15 more

[Error - 20:57:01] 27 Feb 2019, 20:57:01 Error in calling delegate command handler
cannot initialize module FileTabCharacter - Unable to instantiate 'FileTabCharacter' class, it is also not possible to instantiate it as .FileTabCharacter, FileTabCharacterCheck, .FileTabCharacterCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly.
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module FileTabCharacter - Unable to instantiate 'FileTabCharacter' class, it is also not possible to instantiate it as .FileTabCharacter, FileTabCharacterCheck, .FileTabCharacterCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly.
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:463)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:198)
	at com.shengchen.checkstyle.runner.CheckstyleRunner.check(CheckstyleRunner.java:75)
	at com.shengchen.checkstyle.runner.handler.DelegateCommandHandler.executeCommand(DelegateCommandHandler.java:37)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler$1.run(WorkspaceExecuteCommandHandler.java:151)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler.executeCommand(WorkspaceExecuteCommandHandler.java:141)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$4(JDTLanguageServer.java:451)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$26(JDTLanguageServer.java:793)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:479)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Unable to instantiate 'FileTabCharacter' class, it is also not possible to instantiate it as .FileTabCharacter, FileTabCharacterCheck, .FileTabCharacterCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly.
	at com.puppycrawl.tools.checkstyle.PackageObjectFactory.createModule(PackageObjectFactory.java:210)
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:453)
	... 15 more
```

--- a/jdtls.ext/com.shengchen.checkstyle.runner/src/main/java/com/shengchen/checkstyle/runner/CheckstyleRunner.java
+++ b/jdtls.ext/com.shengchen.checkstyle.runner/src/main/java/com/shengchen/checkstyle/runner/CheckstyleRunner.java
@@ -67,7 +67,7 @@ public class CheckstyleRunner {
         // names, see
         // https://sourceforge.net/tracker/?func=detail&aid=2880044&group_id=80344&atid=559497
         checker.setBasedir(null);
-        checker.setModuleClassLoader(Thread.currentThread().getContextClassLoader());
+        checker.setModuleClassLoader(Checker.class.getClassLoader());
         final Properties checkstyleProperties = new Properties();
         checkstyleProperties.putAll(properties);
         final Configuration configuration = ConfigurationLoader.loadConfiguration(configurationFsPath,

